### PR TITLE
Improved support for numbers

### PR DESCRIPTION
**Pull request by theonelucas**

### CODE REVIEW

File: src/components/UiFormInputs/UiInputText/UiInputText.vue [hunk 1]
```diff
@@ -36,6 +36,8 @@
                     :icon="icon"
                     :minlength="min"
                     :maxlength="max"
+                    :min="minsize"
```
Comment (re2005) [inline src/components/UiFormInputs/UiInputText/UiInputText.vue]: minSize should be camel case ?

Comment (re2005) [reply src/components/UiFormInputs/UiInputText/UiInputText.vue]: If you copy the UiInputText from Firm24 it already have all that:

https://github.com/legalthings/firm24/blob/master/src/components/Global/UiFormInputs/UiInputText/UiInputText.vue

File: src/dependencies/i18n.messages.ts [hunk 2]
```diff
@@ -38,6 +39,7 @@ const messages = {
             MIN_LENGTH_AMOUNT: 'Dit veld moet minimaal {length} karakters bevatten',
             NO_ZERO: 'Mag niet nul zijn',
             VALID_CURRENCY: 'Gebruik geldige valutanotatie',
+            MAX_NUMBER_AMOUNT: 'Max {size}',
```
Comment (Sinadinakis) [inline src/dependencies/i18n.messages.ts]: there is a different key for the two translations `MAX_LENGTH_AMOUNT` and `MAX_NUMBER_AMOUNT`. Is this a different key? Why is added only in one language not in both

**Review by re2005 — APPROVED** [review]

File: src/dependencies/i18n.messages.ts (hunk 2 above)
Comment (theonelucas) [reply src/dependencies/i18n.messages.ts]: Correct, I did rename the property, forgot to rename in NL

File: src/components/UiFormInputs/UiInputText/UiInputText.vue (hunk 1 above)
Comment (theonelucas) [reply src/components/UiFormInputs/UiInputText/UiInputText.vue]: Fixed

**Review by Sinadinakis — APPROVED** [review]

File: src/dependencies/i18n.messages.ts [hunk 3]
```diff
@@ -38,6 +39,7 @@ const messages = {
             MIN_LENGTH_AMOUNT: 'Dit veld moet minimaal {length} karakters bevatten',
             NO_ZERO: 'Mag niet nul zijn',
             VALID_CURRENCY: 'Gebruik geldige valutanotatie',
+            MAX_LENGTH_AMOUNT: 'Max {size}',
```
Comment (Sinadinakis) [inline src/dependencies/i18n.messages.ts]: change also the `size` to `length`

Comment (theonelucas) [reply src/dependencies/i18n.messages.ts]: done

**Pull request merged**